CRP.Core/Domain/FinancialAccount.cs

```diff
@@ -49,6 +49,7 @@ public FinancialAccount()
         public virtual Unit Unit { get; set; }
 
         public virtual bool IsActive { get; set; }
+        public virtual bool UserAdded { get; set; }
```

Comment:
@jSylvestre you probably knew I was going to nitpick this but if you have `IsActive` as a bool then this one should be `IsUserAdded` to follow the convention and make it more obvious it's a bool prop